src/tests/queries/tags/delete.test.js

```diff
@@ -0,0 +1,24 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import chai from 'chai';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import chaiHttp from 'chai-http';
+import { Tag } from '../../../queries';
+import * as Factory from '../../../helpers/factory';
+import db from '../../../models';
+
+const article = Factory.article.build();
+delete article.id;
+chai.use(chaiHttp);
+describe('Tag query', () => {
+  it('should update tag', async () => {
+    const action = 'detele';
+    const findArticle = await db.Article.findAll({ limit: 1, logging: false });
```

Comment:
And here

Reply:
With **findAll**, it is easy to return one format of the response to the controller